app/api/event_invoices.py

```diff
@@ -71,6 +74,21 @@ class Meta:
                                   type_='discount-code')
 
 
+class EventInvoiceListPost(ResourceList):
+    """
+    List and create Tracks
+    """
+    def before_post(self, args, kwargs, data):
+        require_relationship(['event'], data)
```

Comment:
I think user is also required as a relation, @SaptakS @shubham-padia please confirm